brooklyn-tosca-transformer/src/main/java/io/cloudsoft/tosca/a4c/brooklyn/spec/StandardInterfaceLifecycleModifier.java

```diff
@@ -34,17 +43,17 @@ public void apply(EntitySpec<?> entitySpec, String nodeId, ToscaApplication tosc
         }
         // If the entity spec is of type VanillaSoftwareProcess, we assume that it's running. The operations should
         // then take care of setting up the correct scripts.
-        entitySpec.configure(VanillaSoftwareProcess.LAUNCH_COMMAND, "true");
-        entitySpec.configure(VanillaSoftwareProcess.STOP_COMMAND, "true");
-        entitySpec.configure(VanillaSoftwareProcess.CHECK_RUNNING_COMMAND, "true");
+        entitySpec.configure(VanillaSoftwareProcess.LAUNCH_COMMAND.getName(), "true");
```

Comment:
Can you expand on the latter two comments? Why can't `DeferredSupplier` be used subsequently? 

Add a comment beforehand to make this clear.